src/octo/core.py

```
@@ -153,3 +153,22 @@ async def cancel(self) -> None:
                     pass
                 case 409:
                     raise CannotPrint()
+
+    async def delete_printed_file(self, file_path: str) -> None:
+        """
+        Deletes a file from the OctoPrint server.
+
+        @param file_path: str The path of the file to delete.
+        """
+        url = f"/api/files/local/{file_path}"
+        async with self.session.delete(url) as resp:
+            self.logger.info(f"delete-printed-file {resp.status}")
+            match resp.status:
+                case 204:
+                    self.logger.info("File deleted successfully")
+                case 404:
+                    self.logger.error("File not found")
```

Should raise an error when status code is not 204, otherwise caller will never be aware that an error was occurred.